src/main/java/frc/robot/commands/drivetrain/drive.java

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.drivetrain;
+
+import java.util.function.Supplier;
+
+import com.ctre.phoenix6.mechanisms.swerve.SwerveModule.DriveRequestType;
+import com.ctre.phoenix6.mechanisms.swerve.SwerveRequest;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.RobotContainer;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.constants.drivetrain.DrivetrainConstants;
+
+public class drive extends Command
+{
+  private Supplier<Double> SPEED_X_SUPPLIER, SPEED_Y_SUPPLIER, ROTATIONAL_SPEED_SUPPLIER;
+  private Drivetrain drivetrain = RobotContainer.drivetrain;
+  
+  /**
+   * @brief Sends a field centric request to the swerve drivetrain with given X speed, Y speed, and rotational speed
+   * @param SPEED_X_SUPPLIER The speed in the X direction
+   * @param SPEED_Y_SUPPLIER The speed in the Y direction
+   * @param ROTATIONAL_SPEED_SUPPLIER The rotational speed
+   */
+  public drive(Supplier<Double> SPEED_X_SUPPLIER, Supplier<Double> SPEED_Y_SUPPLIER, Supplier<Double> ROTATIONAL_SPEED_SUPPLIER) {
+    this.SPEED_X_SUPPLIER = SPEED_X_SUPPLIER;
+    this.SPEED_Y_SUPPLIER = SPEED_Y_SUPPLIER;
+    this.ROTATIONAL_SPEED_SUPPLIER = ROTATIONAL_SPEED_SUPPLIER;
+    addRequirements(drivetrain);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
```

Comment:
Fix the brackets here and whenever we have an empty method we should create a comment that looks like this

public void initialize()
{
   // Intentionally Empty
}